doc/1/controllers/collection/get-mapping/index.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
I think we should keep `getMapping`: renaming the signature will be inconsistent with the restm since others SDK and Kuzzle itself name it `getMapping`

Reply:
Oh, maybe you didnt mean for the function name?